pom.xml

```diff
@@ -64,6 +65,50 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <!-- overlay vivo home files -->
+            <plugin>
+                <artifactId>maven-resources-plugin</artifactId>
```

Comment:
Please add a `<version>` element. See build warning.

Reply:
Will do.

Reply:
Interesting the other plugins are inheriting the version from parent project. This plugin must not be managed in the parent pom. Anyhow, no excuses for missing the warning. -1 point me.